src/Twilio.TaskRouter/Model/Filter.cs

```diff
@@ -17,6 +17,12 @@ public Filter()
         [DataMember(EmitDefaultValue = false, Name="friendly_name")]
         public string FriendlyName { get; set; }
 
+        [DataMember(EmitDefaultValue = false, Name = "filter_friendly_name")]
+        public string FilterFriendlyName {
+            get { return null; }

```

Comment:
you could potentially try callbacks as outlined here: https://blogs.msdn.microsoft.com/carlosfigueira/2011/09/05/wcf-extensibility-serialization-callbacks/

another thought: what happens if you do not expose a getter?